toolchains/docker/toolchain.bzl

```diff
@@ -38,21 +40,35 @@ docker_toolchain = rule(
         "tool_path": attr.string(
             doc = "Path to the docker binary",
         ),
+        "client_config": attr.string(
+            default = "",
+            doc = "A custom directory for the docker client config.json",
+        ),
     },
 )
 
 def _toolchain_configure_impl(repository_ctx):
     tool_path = repository_ctx.which("docker")
+
+    # client_config could be None
```

Comment:
If client_config is not set we need to pass an empty string to the template

Reply:
I'm ensuring that's the case by doing repository_ctx.attr.client_config or "" right?